pkgs/by-name/li/libftdi1/package.nix

```diff
@@ -90,6 +90,10 @@ stdenv.mkDerivation {
       cp -r doc/html "$out/share/doc/libftdi1/"
     '';
 
+  preFixup = ''
+    substituteInPlace $out/lib/pkgconfig/libftdi1.pc --replace-fail "libdir=${placeholder "out"}/${placeholder "out"}/lib" "libdir=${placeholder "out"}/lib"
+  '';
+
```

Comment:
This is can be fixed upstream.

Reply:
I'm sure there's a nixpkgs-specific thing for cmake I'm doing wrong here, but I don't see what upstream is doing wrong here. This is what libftdi1 has in it's CMakeLists.txt:
```
    set(libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
```

where the cmake docs have `CMAKE_INSTALL_PREFIX` conventionally set to /usr/local and `CMAKE_INSTALL_LIBDIR` conventionally set to `lib`. It just so happens that in the cmake setup hook, `CMAKE_INSTALL_PREFIX` is set to a nix output path, as is `CMAKE_INSTALL_LIBDIR`, so we end up with an invalid path that looks like `/nix/store/.../nix/store/...`. I think the onus is on nixpkgs to make the installation work, not libftdi1. Is there another way you think this problem could be solved?

Reply:
@bjornfor do you have any desired changes to make for the libftdi1 change or are you ok with the state of it for now (pending the resolution of that cmake nixpkgs issue)?